models/users.models.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+exports.getUsersMod = () => {
+    let sqlQuery = `SELECT *
+    FROM users `;
+  const queryValues = [];
+
+  return db.query(sqlQuery, queryValues)
+  .then(({ rows }) => {
+    if (rows.length === 0){
+        throw { status: 404, msg: 'Article not found' }  
```

Comment:
Promise.reject*, also not needed?
This endpoint is not relating to articles either.